test/parallel/test-http-request-arguments.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+// test providing both a url and options, with the options partially
```

Comment:
extreme nit: start comments with uppercase ... `// Test providing ...`

(we're *trying* to be more consistent throughout on this)

Reply:
And we also try to add a period to the comment end)